src/chem/airnow2ioda-nc.py

```diff
@@ -127,31 +127,31 @@ def add_data(infile, sitefile):
     loc_mdata = defaultdict(lambda: DefaultOrderedDict(OrderedDict))
     var_mdata = defaultdict(lambda: DefaultOrderedDict(OrderedDict))
     units = {}
-    units['pm25_tot'] = 'microgram/m3'
+    units['pm25'] = 'microgram/m3'
     units['o3'] = 'ppmV'
 
-    for i in ['pm25_tot', 'o3']:
+    for i in ['pm25', 'o3']:
         varDict[i]['valKey'] = i, writer.OvalName()
         varDict[i]['errKey'] = i, writer.OerrName()
         varDict[i]['qcKey'] = i, writer.OqcName()
 
     d = np.empty([nlocs], 'S20')
-    d[:] = f3.time[1].strftime('%Y-%m-%dT%H:%M:%SZ')
+    d[:] = np.array(f3.time[1].strftime('%Y-%m-%dT%H:%M:%SZ'))
     loc_mdata['datetime'] = writer.FillNcVector(d, 'datetime')
     loc_mdata['latitude'] = np.array(f3['latitude'])
     loc_mdata['longitude'] = np.array(f3['longitude'])
     loc_mdata['height'] = np.full((nlocs), 10.)
-    loc_mdata['station_elevation'] = np.full((nlocs), 10.)
+    loc_mdata['station_elevation'] = np.array(f3['elevation'])
 
-    c = np.empty([nlocs], dtype=str)
+    c = np.empty([nlocs], dtype='S20')
     c[:] = np.array(f3.siteid)
     loc_mdata['station_id'] = writer.FillNcVector(c, 'string')
 
-    outdata[varDict['pm25_tot']['valKey']] = np.array(f3['PM2.5'].fillna(nc.default_fillvals['f4']))
+    outdata[varDict['pm25']['valKey']] = np.array(f3['PM2.5'].fillna(nc.default_fillvals['f4']))
     outdata[varDict['o3']['valKey']] = np.array((f3['OZONE']/1000).fillna(nc.default_fillvals['f4']))
-    for i in ['pm25_tot', 'o3']:
+    for i in ['pm25', 'o3']:
         outdata[varDict[i]['errKey']] = np.full((nlocs), 0.1)
-        outdata[varDict[i]['qcKey']] = np.full((nlocs), 0)
+        outdata[varDict[i]['qcKey']] = np.full((nlocs), 1)
```

Comment:
Generally, a QC value of 1 mean that it is a 'bad' observation...

Reply:
Thank you. I changed back to 0